### PR TITLE
Enable one-line installation and smooth onboarding experience

### PR DESCRIPTION
**Pull request by lafh**
Now users can try out HNC by saying `kubectl apply -f <some single
file>`.

### CODE REVIEW

File: incubator/hnc/Makefile [hunk 1]
```diff
@@ -158,3 +158,10 @@ kind-reset: kind-reboot deploy-prereq
 # Convenience target to deploy specifically for kind
 kind-deploy:
 	CONFIG=kind $(MAKE) deploy
+
+###################### RELEASE ACTIONS #########################
+
+# Generate release YAML that can easily install HNC via `kubectl apply -f`
+release: manifests
+	mkdir -p releases
+	kustomize build config/default > releases/manifests.yaml
```
Comment (adrianludwin) [inline incubator/hnc/Makefile]: Why create this new directory? We already have manifests/hnc-manager.yaml, right?

Comment (rjbez17) [reply incubator/hnc/Makefile]: I was under the impression the `manifests` dir was more of kustomize templates and other manifests/templates whereas this is a 'final' built yaml. Maybe `manifests.yaml` is the best name, but I see some advantage to separating 'built/rendered' outside of manifests. That said, I don't really have a strong enough opinion not to change it to use the `manifests` dir.

Comment (adrianludwin) [reply incubator/hnc/Makefile]: No, the templates are in `config`. Only the _fully rendered_ files to in `manifests` - just like `releases`. We should only have one of them.

Comment (rjbez17) [reply incubator/hnc/Makefile]: That’s totally my fault then, I should have caught that. I’ll open a PR to fix it

**Comment by adrianludwin** [comment]
No worries, I spoke to Laetitia about it and she'll look at a way to
combine them.

**Pull request merged**